docs/appendix.rst

```diff
@@ -524,76 +509,190 @@ To fix this, install the Python "wheel" package::
+limit was modified to allow urllib3 package versions >= 2.0.  These new
+versions of the 'urllib3' package include backward incompatible changes
+including:
 
-This also happened after an upgrade of the client OS to Debian buster
+* Support limiting the minimum TLS version to >= 1.2 (i.e OpenSSL version >= 1.1.1).
+  If the version of OpenSSL is less than 1.1.1, this SSLError the error
+  :ref:`ImportError urllib3 v2.0 only supports OpenSSL 1.1.1+` will occur with
+  the initial request to the WBEM Server.
+* Limit the SSL library implementations to just OpenSSL and some versions of
+  LibreSSL.  The  'urllib3' version 2.0+  implementation initial release does
+  not support any other implementations of SSL.
+
+This issue also happens after an upgrade of the client OS to Debian buster
 using Python 3.7, with OpenSSL 1.1.1d.
 
-This is an error that is created by the OpenSSL library and handed back up to
-the SSL module of Python which hands it up to pywbem. The error indicates that
-OpenSSL and the WBEM server do not agree about which SSL/TLS protocol level to
-use.
+This is an error that is created by the SSL library (normally OpenSSL or
+LibreSSL) and handed back up to the SSL module of Python which hands it up to
+pywbem. The error indicates that OpenSSL and the WBEM server do not agree about
+which SSL/TLS protocol level to use.
 
 Pywbem specifies SSL parameters such that the highest SSL/TLS protocol version
-is used that both the client and server support. Thus, pywbem does not put any
-additional restrictions on top of OpenSSL.
+is used that both the client and server support. Pywbem itself does not put any
+additional TLS version restrictions on top of SSL library.
 
 Debian buster includes OpenSSL 1.1.1d and increased its security settings to
-require at least TLS 1.2 (see https://stackoverflow.com/a/53065682/1424462).
-
-This error means that the WBEM server side does not yet support
-TLS 1.2 or higher or that the SSL library used by pywbem does not support
-TL 1.2.
+require at least TLS 1.2 (see `https://stackoverflow.com/a/53065682/1424462`).
 
-This issue can be corrected by:
+This issue can possibly be corrected by:
 
-1. If the current version of urllib3 is less than 2.0, update the version
-   of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy eager urllib3``). Since
-   the pywbem install does not force an eager update of packages, if a valid
-   previous version of urllib3 exists it will not be upgraded to 2.0+ in the
-   reinstallation of pywbem.
+1. If the current version of urllib3 is less than 2.0 (``pip list``), update
+   the version of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy
+   eager urllib3``). to a later version. This is required because pip install
+   does not force an eager update of packages; if a valid previous version (
+   i.e. any version in the range defined by the dev-requirements.txt file) of
+   urllib3 exists it will not be upgraded to 2.0+ in the reinstallation of
+   pywbem.
```

Comment:
Why does the upgrade of urllib3 solve a TLS version mismatch? Is it not rather the upgrade of the SSL library?